src/turn_integrator.py

```diff
@@ -3,20 +3,21 @@
 import requests
 import json
 
-### Load and evaluate the cedentials from the turn_config.json file
-def load_credentials(file_name, line_name):
+## SETUP
+## Load and evaluate the cedentials from the turn_config.json file
+def load_credentials(file_name: str, line_name: str) -> str:
     with open(file_name, 'r') as file:
         turn_config = json.load(file)
 
     return turn_config['lines'][line_name]
 
-def eval_credentials(config_json):
+def eval_credentials(config_json: json) -> str:
     with open('turn_config.json', 'r') as file:
         turn_config = json.load(file)
 
     token_expiry = datetime.strptime(config_json["expiry"], '%b %d, %Y %I:%M %p')
     if token_expiry > datetime.now():
-        return config_json["api_key"]
+        return config_json["token"]
     else:
         raise Exception("API key as expired for this Turn line.")
```

Comment:
I think you mean "API key has expired".

I think maybe we can use a `ValueError` for this instead of the default Exception type?

Reply:
Thanks, done.